src/config/mod.rs

```diff
@@ -16,17 +19,26 @@ pub use jailer::*;
 pub use machine::*;
 use uuid::Uuid;
 
+use crate::Error;
+
+// FIXME: Hardcoding for now. This should come from ChrootStrategy enum, when we've that.
+const KERNEL_IMAGE_FILENAME: &str = "kernel";
+
 /// VMM configuration.
 #[derive(Debug)]
 pub struct Config<'c> {
     pub(crate) socket_path: Cow<'c, Path>,
+    pub(crate) host_socket_path: Cow<'c, Path>,
     log_path: Option<Cow<'c, Path>>,
     log_fifo: Option<Cow<'c, Path>>,
     log_level: Option<LogLevel>,
     metrics_path: Option<Cow<'c, Path>>,
     metrics_fifo: Option<Cow<'c, Path>>,
+    pub(crate) jailer_workspace_dir: Cow<'c, Path>,
     pub(crate) kernel_image_path: Cow<'c, Path>,
+    pub(crate) host_kernel_image_path: Cow<'c, Path>,
     pub(crate) initrd_path: Option<Cow<'c, Path>>,
+    pub(crate) host_initrd_path: Option<Cow<'c, Path>>,
```

Comment:
hmm.. I wonder if the ones we're creating ourselves in the `build` method, need to be Cow'ed. No action needed here but I'll put it on my todo to check..